koncur-tackle-hub/check_images.sh

```
@@ -218,6 +237,9 @@ if [ ${#MISSING[@]} -gt 0 ]; then
                 if [[ "$img" =~ $c_sharp_provider_image_regex ]]; then
                     echo "CSHARP_PROVIDER_IMG=$NEW_TAG" >> $GITHUB_ENV
                 fi
+                if [[ "$img" =~ $generic_provider_image_regex ]]; then
+                    echo "GENERIC_PROVIDER_IMG=$NEW_TAG" >> $GITHUB_ENV
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)